src/Helper.js

```diff
@@ -355,7 +356,15 @@ class Helper {
 
     // Get states whose entity-id starts with the given string.
     const stateEntities = Object.values(this.#hassStates).filter(
-        state => state.entity_id.startsWith(`${domain}.`),
+        state => {
+          if (deviceClass
+            && state.attributes.device_class !== deviceClass
```

Comment:
Unresolved variable device_class.

It should be included in typedef `stateObject` in file `src/typedefs.js`